style-login.css

```diff
@@ -58,10 +60,10 @@ form#loginform p.galogin {
   text-align: center;
   vertical-align: middle;
   border-radius: 0px;
-  padding: 4px;
-  height: 27px;
+  padding: 0;
+  margin: 0;
+  height: auto;
   font-size: 14px;
-  margin-bottom: 6px;
```

Comment:
Check the design from above. We actually need some margin between the button and the "powered" text.

Reply:
I'd removed the 'powered by' text entirely from local test copy, pending questions about whether it's actually required.

Reverted change for now.